include/zephyr/dt-bindings/clock/mspm0_clock.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2025 Texas Instruments Inc.
+ * Copyright 2025 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_MSPM0_CLOCK_H
+#define ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_MSPM0_CLOCK_H
+
+#define MSPM0_CLOCK(bus, bit) ((bus << 8) | bit)
+
+/* Peripheral clock source selection register mask */
+#define MSPM0_CLOCK_PERIPH_REG_MASK(X)	(X & 0xFF)
+
+/* Clock bus references */
```

Comment:
This was already present in previous PR, and is wrong imo. Clocks are used to clock various buses (i2c, spi, uart, ...) but not only buses (timers are not buses afaik). Relevantly, the datasheet never talks about "clock bus" anywhere. Just remove any reference to the word "bus", as well is the macros below

Reply:
ok, will remove the term "bus"